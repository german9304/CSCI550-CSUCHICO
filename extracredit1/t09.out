26 46 599 378
35 30 1027 682
Sum Red Nova 223230
Sum Star Trace 308970
red_nova_stations: 49
max_weight red nova 965
star_trace_stations: 52
max_weight star trace 1001
total : 53220
total: *10 532200
 AD BD inserted
0 26 46 599 378
0 35 30 1027 682
---------
total_values: 
538568
543152
539580
