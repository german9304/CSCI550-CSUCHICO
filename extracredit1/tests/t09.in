49
0 34 784
1 14 879
1 45 292
2 18 504
2 26 489
2 35 479
3 38 436
4 10 965
5 24 418
5 38 549
5 40 306
6 21 207
7 13 832
8 41 864
9 21 578
9 32 883
9 39 911
10 48 248
11 45 559
12 25 326
12 36 673
12 37 594
13 15 268
13 29 482
13 47 535
14 34 789
15 20 825
15 39 736
16 24 228
16 30 411
17 28 584
18 24 344
18 41 270
19 27 208
19 38 180
21 43 91
22 36 667
22 38 596
23 39 36
28 37 65
29 44 125
30 48 322
31 43 604
33 48 290
34 47 166
38 46 162
42 48 505
47 48 58
52
0 22 181
1 14 59
1 27 732
2 13 495
2 18 956
2 30 958
2 43 73
3 12 635
3 33 337
4 31 832
5 45 857
6 15 664
7 34 942
8 33 313
9 38 367
10 20 906
10 49 785
11 43 1001
13 26 228
13 27 603
13 39 994
14 36 662
15 38 369
15 41 728
16 20 785
16 41 472
17 23 304
19 22 503
20 51 772
21 32 813
21 42 573
22 33 285
23 29 852
23 46 637
23 51 760
24 51 140
25 48 928
27 45 813
28 51 107
31 41 597
32 37 625
32 48 944
33 34 242
33 49 899
35 46 237
37 40 589
37 45 785
39 41 354
43 47 922
44 45 345
50 51 937
26 46 599 378
35 30 1027 682
