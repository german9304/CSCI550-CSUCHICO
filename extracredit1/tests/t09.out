26 46 538568
